packages/icon/src/Icon.spec.tsx

```diff
@@ -148,3 +149,9 @@ describe('packages/Icon/createIconComponent', () => {
     expect(glyph.textContent).toBe(text);
   });
 });
+
+describe('Generated icons', () => {
```

Comment:
Would change this to generated glyphs as I believe @DesignerDave made a distinction between the icon component itself and glyphs being the individual exports.